### PR TITLE
updating scenario configs for eval 3

### PR DESCRIPTION
**Pull request by lcadalzo**
- updates scenario configs for eval 3 (add new configs, modify existing parameters, etc.)
- enables random patch location for patch attacks

### CODE REVIEW

**Comment by lcadalzo** [comment]
Link to CI tests. A few of these were failing for `yusong-tan:update_scenario_configs_2` in #1118, so we'd expect they'll fail again here.

**Comment by lcadalzo** [comment]
[Link](https://github.com/lcadalzo/armory/actions/runs/1009316319) to CI tests. A few of these were failing for `yusong-tan:update_scenario_configs_2` in #1118, so they may fail again here.

File: scenario_configs/xview_frcnn_robust_dpatch_defended.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    "adhoc": null,
+    "attack": {
+        "knowledge": "white",
+        "kwargs": {
+            "batch_size": 1,
+            "brightness_range": [
+                0.8,
+                1.0
+            ],
+            "learning_rate": 0.01,
```
Comment (lcadalzo) [inline scenario_configs/xview_frcnn_robust_dpatch_defended.json:12]: @yusong-tan I think there's a minor discrepancy here between the `"learning_rate"` and `"max_iter"` compared to the T&E Eval 3 doc. The latter has:

```
"learning_rate": 0.02,
"max_iter": 100,
```

Is this intentional?

Comment (yusong-tan) [reply scenario_configs/xview_frcnn_robust_dpatch_defended.json:12]: @lcadalzo Yes. In the document, I updated the parameters to match those that seem to work well during the baseline evaluations.

**Review by davidslater — APPROVED** [review]

**Pull request merged**